### PR TITLE
Add login component and functionality

### PR DESCRIPTION
**Pull request by IncredibleBlackMan**
- Add login component and functionality

### CODE REVIEW

File: src/components/loginComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import Loader from './Loader';
+import logo from '../assets/img/logo.png';
+
+class loginComponent extends Component {
+    state = {
+        email: '',
+        password: '',
+    }
+    
+    handleChange = (event) => {
```
Comment (lukorito) [inline src/components/loginComponent.jsx]: Maybe use hooks?

File: src/components/loginComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        const { login } = this.props;
+        login(this.state)
+    }
+
+    render() {
+        const { loading, success, error, data } = this.props;
+        const { email, password } = this.state;
+
+        if (success) {
+            localStorage.setItem('user', JSON.stringify(data));
```
Comment (lukorito) [inline src/components/loginComponent.jsx]: It is not advisable to handle side effects inside components like this. Redux should handle the side effects. I would suggest have a redux middleware the sets the user in the local storage once the action succeeds. 
https://redux.js.org/advanced/middleware
```
const authMiddleware = () => next => (action) => {
  if (!action) return;
  if (action.type === `${types.LOGIN}_SUCCESS`) {
    const payloadData = action.payload.data;
    const { token } = payloadData;
    action.payload.token = token;
    localStorage.setItem('token', token);
  }
  if (action.type === types.LOGOUT) {
    localStorage.removeItem('token');
  }
  return next(action);
};
```

File: src/components/loginComponent.jsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        }
+        return(
+            <div id="body" className="login-body">
+                <div className="form-content">
+                    {success ? <Redirect to="/login.html" /> : null}
```
Comment (lukorito) [inline src/components/loginComponent.jsx]: `/login.html` is that the actual path?

File: src/components/loginComponent.jsx (hunk 2 above)
Comment (IncredibleBlackMan) [reply src/components/loginComponent.jsx]: On it.

File: src/components/loginComponent.jsx (hunk 3 above)
Comment (IncredibleBlackMan) [reply src/components/loginComponent.jsx]: No. I am yet to work on the component to redirect to.

**Review by lukorito — APPROVED** [review]

**Pull request merged**